sdks/java/io/iceberg/src/test/java/org/apache/beam/sdk/io/iceberg/catalog/IcebergCatalogBaseIT.java

```diff
@@ -315,7 +319,9 @@ private List<Record> readRecords(Table table) {
     return writtenRecords;
   }
 
-  @Test
+  private static final int FIVE_MINUTES_IN_MS = 5 * 60 * 1000;
+
+  @Test(timeout = FIVE_MINUTES_IN_MS)
```

Comment:
We can use a `@Rule` to set timeout for all tests, which has a smaller diff:

```
@Rule public transient Timeout globalTimeout = Timeout.seconds(300);
```

Reply:
That's great, thanks for the tip!

Reply:
Done